lib/src/modules/task.cc

```diff
@@ -426,9 +438,110 @@ static fs::path getCachedTaskFile(const fs::path& task_cache_dir,
     }
 }
 
+// build a spool dir for the supporting library files requested by a multifile task
+static fs::path createInstallDir(const fs::path& spool_dir, const std::set<std::string>& download_set) {
+    auto install_dir = spool_dir / fs::unique_path("temp_task_%%%%-%%%%-%%%%-%%%%");
```

Comment:
We should add some debug logging about this to make it easy to find the installdir for a past run (in the logs).

Reply:
`LOG_INFO()`? or `TRACE`? or ?

Reply:
LOG_DEBUG